misk/src/main/kotlin/misk/cloud/fake/security/keys/FakeKeyManager.kt

```diff
@@ -0,0 +1,33 @@
+package misk.cloud.fake.security.keys
+
+import misk.security.keys.KeyManager
+import okio.ByteString
+import java.nio.ByteBuffer
+import java.security.Key
+import javax.crypto.Cipher
+import javax.crypto.spec.SecretKeySpec
+
+internal class FakeKeyManager : KeyManager {
+    override fun encrypt(keyAlias: String, plainText: ByteString): ByteString {
+        val key = newSymmetricKey(keyAlias)
+        val cipher = Cipher.getInstance("AES")
+        cipher.init(Cipher.ENCRYPT_MODE, key)
+        return ByteString.of(ByteBuffer.wrap(cipher.doFinal(plainText.toByteArray())))
+    }
+
+    override fun decrypt(keyAlias: String, cipherText: ByteString): ByteString {
+        val key = newSymmetricKey(keyAlias)
+        val cipher = Cipher.getInstance("AES")
+        cipher.init(Cipher.DECRYPT_MODE, key)
+        return ByteString.of(ByteBuffer.wrap(cipher.doFinal(cipherText.toByteArray())))
+    }
+
+    private fun newSymmetricKey(keyText: String): Key {
+        val keyInput =
+                if (keyText.length > 32) keyText.take(32)
```

Comment:
This might be better as a `when` statement?

Reply:
when isn't really great for comparison operators, but can do it with a range